### ISSUE TITLE
Delimited Text - Preview only shows 2 lines

### ISSUE THREAD

**Issue by ArkitectNaut** [qgis/QGIS#51019]
### What is the bug or the crash?

When importing a .csv file in the Delimited Text view, I am only shown a 2 line preview of a 20 entry database. It seems like the size of this preview table is not adjustable/resizable. This makes the process of checking that all entries are properly formatted very tedious.

### Steps to reproduce the issue

1. Go to _Data Source Manager_ > _Delimited Text_
2. Choose a .csv file
3. View preview table

### Versions

QGIS version
3.22.13-Białowieża
QGIS code revision
02ff117c
Qt version
5.15.3
Python version
3.9.5
GDAL/OGR version
3.6.0
PROJ version
9.1.0
EPSG Registry database version
v10.074 (2022-08-01)
GEOS version
3.11.1-CAPI-1.17.1
SQLite version
3.39.4
PDAL version
2.4.3
PostgreSQL client version
14.3
SpatiaLite version
5.0.1
QWT version
6.1.6
QScintilla2 version
2.13.1
OS version
Windows 10 Version 2009

Active Python plugins
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.5
processing
2.12.99
sagaprovider
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [x] I tried with a new QGIS profile

### Additional context

I tried to resize the whole window but the preview remains only two lines tall.

**Comment by agiudiceandrea** [comment]
@ArkitectNaut could you please provide a sample csv file in order to reproduce the issue?

**Comment by ArkitectNaut** [comment]
![Delimited_Text](https://user-images.githubusercontent.com/92726010/205489724-b249597b-48c2-4a4f-bf52-86ea7014fbd7.png)
[Index_Photo_Survey.csv](https://github.com/qgis/QGIS/files/10148455/Index_Photo_Survey.csv)

**Comment by agiudiceandrea** [comment]
@ArkitectNaut, I don't see any actual issue even using the provided csv file.

https://user-images.githubusercontent.com/16253859/205495808-ec7873dd-9a26-4e10-85f6-07ed4b1e0bb3.mp4

Could you please try using a new QGIS user profile?

**Comment by ArkitectNaut** [comment]
![New_QGIS_Profile](https://user-images.githubusercontent.com/92726010/208415614-360638eb-8a5a-4ac1-9afe-7b50348b7fdf.jpg)

I created a new QGIS profile but nothing changed. The preview table of the data is still only 2 lines tall and I am presented with no option to enlarge this preview table view. I tried expanding the various options, re-sizing the window, re-sizing the side pane, but nothing changes.
